.github/workflows/docker.yml

```
@@ -15,49 +15,91 @@ env:
           push: ${{ github.event_name != 'pull_request' }}
           pull: true
           tags: ${{ steps.docker_meta.outputs.tags }}
           labels: ${{ steps.docker_meta.outputs.labels }}
-      
+          attests: |
+            type=sbom
+            type=provenance,mode=max
+
+      - name: Tag image for PR
+        if: ${{ github.event_name == 'pull_request' }}
+        run: |
```

actionlint (1.7.7)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)